doc/developer/server_side.rst

```diff
@@ -324,16 +319,11 @@ Or in c2cgeoportal root directory:
 .. prompt:: bash
 
     ./docker-compose-run alembic \
-        --config tests/functional/alembic[_static].ini \
+        --config geoportal/tests/functional/alembic[_static].ini \
```

Comment:
If I'm not missing something, it should be ./docker-compose-run alembic --name=[main|static]

Reply:
thanks :-)